features/createUser.feature

```diff
@@ -7,4 +7,5 @@ Feature: Create user
     Given I have a users name
         And I have a users id
     When I create a user
-    Then I receive a user object
+    Then I receive a HTTP 201 code
```

Comment:
Well, I consider these tests as written based on the documentation. Ideally, someone could read the documentation and write tests that will succeed based on the assumptions made while reading the docs.